pandas/tests/io/test_clipboard.py

```diff
@@ -76,10 +76,41 @@ def df(request):
         raise ValueError
 
 
+# our local clipboard for tests
+_mock_data = {}
```

Comment:
it’s really odd to keep state outside of this fixture when it’s a function level

why don’t u just make it a module level?
or it seems that function level might be ok with state inside